src/ui/components/filters/filteroptionscomponent.js

```diff
@@ -360,10 +360,7 @@ export default class FilterOptionsComponent extends Component {
       if (clearSearchEl && searchInputEl) {
         DOM.on(clearSearchEl, 'click', event => {
           searchInputEl.value = '';
-          searchInputEl.dispatchEvent(new Event('input', {
-            'bubbles': true,
-            'cancelable': true
-          }));
+          DOM.trigger(searchInputEl, 'input');
```

Comment:
Do we need to include any of the event settings in the call to `DOM.trigger`? Or do the defaults in `_customEvent` get us what we need?

Reply:
yes the defaults are the same